src/eradius_client.erl

```diff
@@ -62,19 +62,29 @@ send_request(NAS, Request) ->
 %   If no answer is received within the specified timeout, the request will be sent again.
 -spec send_request(nas_address(), #radius_request{}, options()) ->
     {ok, binary(), eradius_lib:authenticator()} | {error, 'timeout' | 'socket_down'}.
-send_request({Host, Port, Secret}, Request, Options) 
+send_request({Host, Port, Secret}, Request, Options)
```

Comment:
please, don't mix pure formatting and white space changes with functional changes.

If you think the whitespace problems need fixing, create a separate pull request for that.